caffe2/python/operator_test/pack_ops_test.py

```diff
@@ -49,6 +53,58 @@ def pack_segments_ref(lengths, data):
 
         return pack_segments_ref
 
+    @given(
+        num_seq=st.integers(10, 100),
+        cell_size=st.integers(1, 10),
+        **hu.gcs
+    )
+    def test_pack_with_max_length_ops(self, num_seq, cell_size, gc, dc):
+        # create data
+        lengths = np.arange(num_seq, dtype=np.int32) + 1
+        num_cell = np.sum(lengths)
+        data = np.zeros(num_cell * cell_size, dtype=np.float32)
+        left = np.cumsum(np.arange(num_seq) * cell_size)
+        right = np.cumsum(lengths * cell_size)
+        for i in range(num_seq):
+            data[left[i]:right[i]] = i + 1.0
+        data.resize(num_cell, cell_size)
+        print("\nnum seq:{},    num cell: {},   cell size:{}\n".format(
+            num_seq, num_cell, cell_size)
+            + "=" * 60
+        )
+        # run test
+        max_length = 100
```

Comment:
`num_seq` can be 100 (since `st.integers` can return value <= max_value), maybe set it to `num_seq + 1`?

Reply:
Yes, I agree.